src/Contract/Operation/Mapable.php

```diff
@@ -15,12 +15,15 @@
 /**
  * @template TKey
  * @template T
- * @template V
  */
 interface Mapable
 {
     /**
      * Apply one or more callbacks to a collection and use the return value.
+     * Usage with multiple callbacks is deprecated and will be removed in a future major version.
+     * Use mapN instead for multiple callbacks.
+     *
+     * @template V
```

Comment:
the template for the return type needs to be on the method, not on the interface, because it comes from the callback passed to the method. That's why there was [this error](https://github.com/loophp/collection/pull/112/files?diff=split&w=1#diff-995edee38ad4f8387e58ebd52c31bcc04c56cc2448d331b1cf5e0b35c57b9efaL9-L12) reported before

Reply:
Yep exactly that!